swagger.yaml

```
@@ -3447,17 +3447,52 @@ paths:
           format: uuid
           required: False
         - in: query
-          name: host_id
+          name: hosts_id
```

might be more English-like
```suggestion
          name: host_ids
```